Fixed LuisModelUrl

csiebler opened the pull request:
Luis expects `?subscription-key` instead of `&subscription-key`, which otherwise results in a 404 error

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

sahaj08 commented:
#sign-off

sahaj08 commented:
#sign-off

rmca14 commented:
@sahaj08 
Please correct the filename by inserting a hyphen where it's needed. Thanks.

@barbkess

sahaj08 commented:
@rmca14 I have corrected the filename.

sahaj08 commented:
#sign-off

sahaj08 commented:
Closing this pull request and creating one in the new azure-content-pr branch

csiebler committed:
Fixed LuisModelUrl

Luis expects `?subscription-key` instead of `&subscription-key`, which otherwise results in a 404 error
